stablehlo/reference/InterpreterOps.cpp

```diff
@@ -30,6 +30,7 @@ limitations under the License.
 #include "mlir/IR/SymbolTable.h"
 #include "mlir/Support/DebugStringHelper.h"
 #include "mlir/Support/LLVM.h"
+#include "stablehlo/dialect/Base.h"
```

Comment:
Is this needed? No file change? Also if it is needed, is there a corresponding bazel dep change?

Reply:
`stablehlo/reference/InterpreterOps.cpp.inc` needs the declaration for isValidStablehloQuantizedElementType which is in Base.h

>> Also if it is needed, is there a corresponding bazel dep change?

You can find that in the  stablehlo/Bazel.build file.